src/App/App.tsx

```diff
@@ -26,6 +14,12 @@ class App extends React.Component<{}, AllData> {
       allPlanets: planetData
     };
   }
+
+  componentDidMount = () => {
+    discoverPlanets()
+      .then(result => this.setState({allPlanets: result}))
+  }
```

Comment:
Later we can implement a catch here for check for server errors

Reply:
I'll make an issue for it now so we don't forget